### PR TITLE
Add -v option and error handling in convert mode

### PR DESCRIPTION
**Pull request by bjk7119**
## Description

- Add -v option to print FOSSLight Reuse version
- Add error handling code in convert mode
    - Print creating file log when only write output_file successfully, or not exit system
- Check file name in convert mode (oss-pkg-info*.yaml file)
- Delete creating .csv file

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_reuse/cli.py [hunk 1]
```diff
@@ -29,6 +30,9 @@ def main():
     if args.help:
         print_help_msg()
 
+    if args.version:
+        print_version(PKG_NAME)
+
```
Comment (soimkim) [inline src/fosslight_reuse/cli.py:35]: `sys.exit(0)` is needed.
or add this to below if statement.

Comment (soimkim) [reply src/fosslight_reuse/cli.py:35]: ex. https://github.com/fosslight/fosslight_scanner/blob/main/src/fosslight_scanner/cli.py
ex2. https://github.com/fosslight/fosslight_dependency_scanner/blob/main/src/fosslight_dependency/run_dependency_scanner.py

File: src/fosslight_reuse/_help.py [hunk 2]
```diff
@@ -36,3 +37,7 @@
 def print_help_msg(exitOpt=True):
     helpMsg = PrintHelpMsg(_HELP_MESSAGE_REUSE)
     helpMsg.print_help_msg(exitOpt)
+
+
+def print_version(pkg_name):
```
Comment (soimkim) [inline src/fosslight_reuse/_help.py:42]: Remove this function and use it directly.

File: src/fosslight_oss_pkg/fosslight_oss_pkg.py [hunk 3]
```diff
@@ -85,13 +85,16 @@ def convert_report(base_path, output_name, format, need_log_file=True, sheet_nam
                     sys.exit(1)
                 convert_excel_mode = True
                 report_to_read = base_path
-            else:
+            elif base_path.endswith((".yaml", ".yml")) and base_path.startswith("oss-pkg-info"):
```
Comment (soimkim) [inline src/fosslight_oss_pkg/fosslight_oss_pkg.py:88]: How about using this?
```
p = re.compile(r"oss-pkg-info[\s\S]*.ya?ml", re.I)
p.search(base_path)
```

Comment (soimkim) [reply src/fosslight_oss_pkg/fosslight_oss_pkg.py:88]: This is to match case-insensitively.

Comment (bjk7119) [reply src/fosslight_oss_pkg/fosslight_oss_pkg.py:88]: It's simple and better than mine

File: src/fosslight_oss_pkg/fosslight_oss_pkg.py [hunk 4]
```diff
@@ -31,8 +32,8 @@ def find_report_file(path_to_find):
         for root, dirs, files in os.walk(path_to_find):
             for file in files:
                 file_name = file.lower()
-                if (file_name.startswith("oss-report") or file_name.startswith("fosslight-report")) \
-                   and file_name.endswith(".xlsx"):
+                p = re.compile(r"[\s\S]*OSS[\s\S]*-Report[\s\S]*.xlsx", re.I)
```
Comment (soimkim) [inline src/fosslight_oss_pkg/fosslight_oss_pkg.py:35]: In this case FOSSLight-Report isn't found

File: src/fosslight_oss_pkg/fosslight_oss_pkg.py [hunk 5]
```diff
@@ -80,18 +81,25 @@ def convert_report(base_path, output_name, format, need_log_file=True, sheet_nam
...
         if base_path != "":
             if base_path.endswith(".xlsx"):
-                if output_extension == '.xlsx':
-                    logger.error("Format error - can make only .yaml file")
-                    sys.exit(1)
-                convert_excel_mode = True
-                report_to_read = base_path
+                p = re.compile(r"[\s\S]*OSS[\s\S]*-Report[\s\S]*.xlsx", re.I)
+                if p.search(base_path):
+                    if output_extension == '.xlsx':
```
Comment (soimkim) [inline src/fosslight_oss_pkg/fosslight_oss_pkg.py:86]: Why this is needed?

**Review by soimkim — APPROVED** [review]

**Pull request merged**